src/main/takeDetection/sentenceSimilarity.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * https://github.com/stephenjjbrown/string-similarity-js
```

Comment:
Have we modified this or is it just a one for one copy of the linked code? If the latter then why not just import as a library

Reply:
@zwan0202